docs/Procedure/Documentation/Step_Registry_Documentation_Guide.md

```diff
@@ -0,0 +1,155 @@
+
+## How to Document a Step in the Step Registry
+All steps in the Step Registry, at least written by the Interop team, should have a `README.md` file associated with it in the same folder as the config and BASH script for that step. Documentation should be written in Markdown. If you are unfamiliar with Markdown, please use [the list of resources](#markdown-resources) at the end of this documentation to familiarize yourself with it.
+
+### Getting Started
+1. Create the `README.md` file in the same directory that holds the configuration file for your new step.
+2. Add a level-1 header to the top of the file with the name of the name of the step. 
+   - **Note:** Please add `<!-- omit from toc -->` to the end of the header to avoid adding it to the Table of Contents.
+   - **Example:** `# interop-tooling-new-step <!-- omit from toc -->` 
+   
+3. Add a level-2 header below the first header with the text "Table of Contents"
+   - **Example:** `## Table of Contents <!-- omit from toc -->`
+4. Add the Table of Contents
+   1. In Visual Studio Code, make sure the [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) plugin is installed
+   2. Navigate to the line below the "Table of Contents" header and press `CTRL+SHIFT+P` to bring up the Command Pallette.
+   3. Type in `Markdown All in One: Create Table of Contents`
+   4. Press enter
+   - **Note:** This plugin will create the Table of Contents for you and keep it automatically updated with hyperlinks upon every save of the document.
+
+### Adding Documentation
+The outline of these documents should generally follow the following structure:
+- Purpose
+- Process
+- Requirements
+  - Variables
+  - Infrastructure
+  - Credentials
+
+If you'd like to expand on this structure, please feel free to. But, this structure should be the minimum for each new step created.
+
+#### Purpose
+This section of your document should outline the reason this step was created and what purpose it serves. This section doesn't need to be more than a paragraph or two, but make sure to mention if this step was created for a specific scenario or to be used by multiple scenarios.
+
+#### Process
+This section of your document should describe how your new step works. You should try to go into a reasonable amount a detail on how the step works. Here are some things you could include:
+
+- If this step is a workflow or chain, link the steps it uses in order and why each step is needed
+- If this step is a ref, outline how the BASH script used in the ref and how it works
```

Comment:
change `outline how the BASH script used` to `outline how the BASH script is used`